esmvaltool/diag_scripts/primavera/eady_growth_rate/eady_growth_rate.py

```diff
@@ -0,0 +1,210 @@
+import logging
+import os
+import sys
+
+import cartopy.crs as ccrs
+import iris
+import iris.analysis
+import iris.cube
+import iris.quickplot as qplt
+import iris.util
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+from dask import array as da
+
+import esmvaltool.diag_scripts.shared
+import esmvaltool.diag_scripts.shared.names as n
```

Comment:
I'd recommend naming this something like `shared_names`:

```suggestion
import esmvaltool.diag_scripts.shared.names as shared_names
```